plugin/trino-pinot/src/main/java/io/trino/plugin/pinot/query/DynamicTable.java

```diff
@@ -134,6 +138,11 @@ public String getQuery()
         return query;
     }
 
+    public boolean isAggregateInSelectList()
```

Comment:
Do I understand correctly that this tracks if the aggregation column is already part of the projected columns or not?

Reply:
Yep, that's correct.

Reply:
Also changed this to `isAggregateInProjections` to be consistent. wdyt?